src/plugins/kconfig/file_utility.hpp

```diff
@@ -35,28 +35,28 @@ class FileUtility
 	/**
 	 * @brief This method is used to check if the next character in the buffer is end of file
 	 *
-	 * @return true if the next character is end of file, false otherwise
+	 * @retval true next character is end of file
```

Comment:
Sorry for not clarifying [my comment](https://github.com/ElektraInitiative/libelektra/pull/3249#discussion_r349597082) further. When I wrote 

> If you want you can use the Doxygen command [`retval`](http://www.doxygen.nl/manual/commands.html#cmdretval) to specify the different return values.

I actually meant that you could use something like:

https://github.com/ElektraInitiative/libelektra/blob/d2169cff88b6003ef35c80852868a59aeed075cc/src/bindings/cpp/include/key.hpp#L1331-L1332

. For example, I think replacing the text with

```
@retval true if the next character is end of file
@retval false otherwise
```

or something similar here would make sense.

Reply:
This was the first example I found, so I was based here:
```
/**
 * @copydoc keyCmp
 *
 * @retval true == 0
 */
```
But it makes sense, I'll add the `@retval` for `false` values as well.
Thanks